recipes/recipes_emscripten/xwidgets/recipe.yaml

```diff
@@ -12,21 +12,23 @@ source:
   - patches/shared.patch
 
 build:
-  number: 4
+  number: 5
 
 requirements:
   build:
   - ${{ compiler("cxx") }}
   - cmake
   - make
   host:
-  - nlohmann_json >=3.11
+  - nlohmann_json >=3.12
   - xeus >=5,<6
   - xproperty >=0.12.0,<0.13
   run:
-  - nlohmann_json >=3.11
+  - nlohmann_json >=3.12
   - xeus >=5,<6
   - xproperty >=0.12.0,<0.13
+  run_exports:
+  - ${{ pin_subpackage("xwidgets", upper_bound="x.x") }}
```

Comment:
I think it should be ` max_pin`  instead of ` upper_bound`  (or mayabe I am not aware of the existenc of `upper_bound`, first time I see it).

Reply:
Ahh I used max_pin initially (look at the commit before this) but the CI asked me to replace it with upper_bound !

Reply:
Oh ok so that's probably from the new recipes format.